tutorials/blender_procedural_datasets.md

```diff
@@ -0,0 +1,109 @@
+\page blender_procedural_datasets Generation of Procedural Datasets with Blender
+
+[url_blender_org]: https://blender.org
+[Blender][url_blender_org] is a free and open-source 3D software suite that
+includes several tools and features for the creation of models that can be
+utilized inside robotics simulators like Gazebo. In addition to modeling,
+sculpting and texturing models manually, Blender also enables procedural
+generation of datasets that can facilitate diversity in simulation environments.
+
+[procedural_dataset_generator_script]: https://github.com/ignitionrobotics/ign-gazebo/tree/ign-gazebo6/examples/scripts/blender/procedural_dataset_generator.py
+Script [`procedural_dataset_generator.py`][procedural_dataset_generator_script]
+supports the generation of procedural SDF models for use inside Ignition Gazebo.
+Its exporting capabilities can be used with a number of existing `.blend` files.
+However, the Blender project must use node-based modifiers to enable the
+generation of procedural datasets (examples shown below). This script has the
+following features:
+
+1. Generation of SDF models compatible with Ignition Fuel.
+
+2. Exporting visual and collision mesh geometry (at different levels of detail).
+
+3. Linking to existing PBR textures via symlink or hard copy.
+
+4. Estimation of inertial properties assuming uniform density.
+
+5. Randomization of parameters, e.g. mass and surface friction.
```

Comment:
Some of these topics are mentioned here but not described in detail in the tutorial. People would have to read the source code to have an idea of what these are referring to. Could you add some sections summarizing what is done in the script in English, so that a reader could have a conceptual idea, without having to look at the source code?

Reply:
I improved this section and also included CLI arguments that provide a more extensive help message via `-h` flag (discussed below).